arch/arm/soc/st_stm32/stm32l0/soc.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ * Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for STM32L0 processor
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+#include <arch/cpu.h>
+#include <cortex_m/exc.h>
+#include <linker/linker-defs.h>
+#include <string.h>
+
+
+/**
+ * @brief Relocate vector table to SRAM.
+ *
+ * On Cortex-M0+ platforms, the Vector Base address cannot be changed.
```

Comment:
AFAIK there is VTOR on Cortex-M0+, so all this relocation stuff may be unneeded on L0.

Reply:
Are you sure it can be removed?

Reply:
It can be removed eventually, but that may require some changes in arch/arm/core/cortex_m/prep_c.c and maybe somethere else. I think @b0661 and @superna9999 can help with that.